src/amo/pages/Home/index.js

```diff
@@ -275,120 +193,47 @@ export class HomeBase extends React.Component {
           {isDesktopSite ? (
             <HeroRecommendation
               errorHandler={errorHandler}
-              shelfData={heroShelves ? heroShelves.primary : undefined}
+              shelfData={homeShelves ? homeShelves.primary : undefined}
             />
           ) : null}
 
           <div className="Home-content">
             {isDesktopSite ? (
               <SecondaryHero
-                shelfData={heroShelves ? heroShelves.secondary : undefined}
+                shelfData={homeShelves ? homeShelves.secondary : undefined}
               />
             ) : null}
 
             {!isDesktopSite ? this.renderHeroHeader() : null}
 
-            <LandingAddonsCard
-              addonInstallSource={INSTALL_SOURCE_FEATURED}
-              addons={shelves.recommendedExtensions}
-              className="Home-RecommendedExtensions"
-              header={i18n.gettext('Recommended extensions')}
-              footerText={i18n.gettext('See more recommended extensions')}
-              footerLink={{
-                pathname: '/search/',
-                query: {
-                  addonType: ADDON_TYPE_EXTENSION,
-                  promoted: RECOMMENDED,
-                  sort: SEARCH_SORT_RANDOM,
-                },
-              }}
-              loading={loading}
-              placeholderCount={
-                isDesktopSite
-                  ? LANDING_PAGE_EXTENSION_COUNT
-                  : MOBILE_HOME_PAGE_EXTENSION_COUNT
-              }
-            />
+            {isDesktopSite ? (
+              <HomepageShelves
+                loading={loading}
+                placeholderCount={LANDING_PAGE_EXTENSION_COUNT}
```

Comment:
It seems that we always pass `LANDING_PAGE_EXTENSION_COUNT` into `placeholderCount`, so we might as well dispense with that prop and just define it in `HomepageShelves`. I mentioned this above as well.